### ISSUE TITLE
Something I thought about (and just ran into). The `%post` should have something about enabling powertools. `dnf config-manager --set-enabled powertools` - I've been testing my pagure role and it blew up because of missing deps... and they're in powertools.

### ISSUE THREAD

**Issue by NeilHanlon** [rocky-linux/infrastructure#14943]
Something I thought about (and just ran into). The `%post` should have something about enabling powertools. `dnf config-manager --set-enabled powertools` - I've been testing my pagure role and it blew up because of missing deps... and they're in powertools.

_Originally posted by @nazunalika in https://github.com/rocky-linux/infrastructure/pull/5#issuecomment-745705688_

Own issue so we can reference it during code review.

**Comment by nazunalika** [comment]
Context: Ansible and other packages we will end up relying on require the EPEL repository. The EPEL repository requires the PowerTools (or codeready for RHEL) repo to be enabled. 

In RHEL 8 it is required to also enable the codeready-builder-for-rhel-8-*-rpms repository since EPEL packages may depend on packages from it:

```
    # ARCH=$( /bin/arch )
    # subscription-manager repos --enable "codeready-builder-for-rhel-8-${ARCH}-rpms"
```
On CentOS 8 it is recommended to also enable the powertools repository since EPEL packages may depend on packages from it:

```
    # dnf config-manager --set-enabled powertools
```

https://fedoraproject.org/wiki/EPEL

**Comment by NeilHanlon** [comment]
@nazunalika should this be in the kickstart?

**Comment by nazunalika** [comment]
This could be in the kickstart or we can address it on a per-playbook basis (this may or may not be problematic, it's something I'd have to think about) - the tools (like MBS, mock, koji) rely on EPEL, and stuff in EPEL depends on packages in powertools.

**Comment by NeilHanlon** [comment]
Makes sense to me. I think having a specific 'role' for epel which
includes our mirrors for epel & configures powertools probably makes the
most sense..

E.g.

- Something I need to setup (koji) needs epel/powertools.
- My module includes the epel 'role' (not really the right word, but
   alas)
- That 'role' configures the host to point to the right mirror
   internally, and sets up powertools as well. 

Basically just bundle them together as 'one' for our purposes.
